src/components/Button/Button.tsx

```diff
@@ -8,61 +8,126 @@ import useMergeRefs from "hooks/useMergeRefs";
 import { NOOP } from "utils/function-utils";
 import Icon from "components/Icon/Icon";
 import Loader from "components/Loader/Loader";
-import { BUTTON_COLORS, BUTTON_INPUT_TYPE, BUTTON_TYPES, getActualSize } from "./ButtonConstants";
+import { ButtonColor, ButtonInputType, ButtonType, getActualSize } from "./ButtonConstants";
 import { getParentBackgroundColorNotTransparent, TRANSPARENT_COLOR } from "./helper/dom-helpers";
 import "./Button.scss";
+import { Rect } from "@popperjs/core";
 
+// @ts-ignore
 const isIE11 = !!window.MSInputMethodContext && !!document.documentMode;
 
 // min button width
 const MIN_BUTTON_HEIGHT_PX = isIE11 ? 32 : 6;
 const UPDATE_CSS_VARIABLES_DEBOUNCE = 200;
 
-const Button = forwardRef(
+export interface ButtonProps {
```

Comment:
I think publisher-components need an update as well, no?

Reply:
True, but this is just a POC, so I kept only the really relevant files 🙂